docs/micro-rdk/motor/gpio.md

```diff
@@ -0,0 +1,153 @@
+---
+title: "Configure a gpio motor"
```

Comment:
So this page is fairly the same as the gpio motor one I assume?

Reply:
It is very similar with a difference in pin configuration options